### PR TITLE
chore: update positioning/messaging

### PR DESCRIPTION
**Pull request by igorlukanin**
https://deploy-preview-4155--cubejs-docs-staging.netlify.app/introduction

### CODE REVIEW

**Comment by keydunov** [comment]
@igorlukanin Do you want to change it iteratively? It is fine to start with this small change, but we need to update the rest of the pages too.

**Review by keydunov — APPROVED** [review]
Good as a first step, not enough though.

**Comment by igorlukanin** [comment]
@keydunov Yep, I'd like this to be a small focused update, and then we can iterate.

File: README.md [hunk 1]
```diff
@@ -57,22 +55,22 @@ You are also welcome to join our **monthly community calls** where we discuss co
 
 ### Contributing
 
-There are many ways you can contribute and help Cube.js. Here a few ones:
+There are many ways you can contribute to Cube. Here a few ones:
```
Comment (rpaik) [inline README.md:58]:
```suggestion
There are many ways you can contribute to Cube, and here are a few possibilities:
```

File: README.md [hunk 2]
```diff
@@ -57,22 +55,22 @@ You are also welcome to join our **monthly community calls** where we discuss co
...
-* Add Cube.js to your stack on [Stackshare](https://stackshare.io/cube-js).
+* Add Cube to your stack on [Stackshare](https://stackshare.io/cube-js).
 * Upvote issues with 👍 reaction so we know what's the demand for particular issue to prioritize it within road map.
 * Create issues every time you feel something is missing or goes wrong.
 * Ask questions on [Stack Overflow with cube.js tag](https://stackoverflow.com/questions/tagged/cube.js) if others can have these questions as well.
-* Provide pull requests for all open issues and especially for those with [help wanted](https://github.com/cube-js/cube.js/issues?q=is%3Aissue+is%3Aopen+label%3A"help+wanted") and [good first issue](https://github.com/cube-js/cube.js/issues?q=is%3Aissue+is%3Aopen+label%3A"good+first+issue") labels as those are out of Cube.js Core Team roadmap scope.
+* Provide pull requests for all open issues and especially for those with [help wanted](https://github.com/cube-js/cube.js/issues?q=is%3Aissue+is%3Aopen+label%3A"help+wanted") and [good first issue](https://github.com/cube-js/cube.js/issues?q=is%3Aissue+is%3Aopen+label%3A"good+first+issue") labels as those are out of Cube team roadmap scope.
```
Comment (rpaik) [inline README.md:65]:
```suggestion
* Provide pull requests for all open issues and especially for those with [help wanted](https://github.com/cube-js/cube.js/issues?q=is%3Aissue+is%3Aopen+label%3A"help+wanted") and [good first issue](https://github.com/cube-js/cube.js/issues?q=is%3Aissue+is%3Aopen+label%3A"good+first+issue") labels.
```

Comment (rpaik) [reply README.md:65]: I think this makes it simpler/clearer.

**Comment by rpaik** [comment]
A couple of minor nits 😄

**Comment by cowperthwait** [comment]
@igorlukanin I’d like to propose a series of small text edits — as a first, immediate step… — but I’m not sure of the best way to submit them. Do you want me to float a PR to these files?

**Comment by keydunov** [comment]
@cowperthwait can you propose all of them here in PR?

**Pull request merged**